### ISSUE TITLE
ICESUGAR target fails to build

### ISSUE THREAD

**Issue by piso77** [BrunoLevy/learn-fpga#39]
learn-fpga/FemtoRV$ make ICESUGAR
...
make[2]: Leaving directory '/home/flag/fpga_stuff/learn-fpga/FemtoRV/FIRMWARE/CRT'
==== Generated femtorv32 libs.
make[1]: Leaving directory '/home/flag/fpga_stuff/learn-fpga/FemtoRV/FIRMWARE'
make: *** No rule to make target 'FIRMWARE/firmware.hex', needed by 'ICESUGAR.synth'.  Stop.

**Comment by BrunoLevy** [comment]
Yes you'll need to adapt icesugar.mk and create RTL/CONFIGS/icesugar_config.v plus a couple of other changes (I could not test it with the icesugar). It is not very difficult to do, let me know if you want to do it / test it and I'll help you.

**Comment by abderraouf-adjal** [comment]
@piso77 Hi, where you able to do a core on icesugar or icesugar-nano on the SPI flash?

**Comment by piso77** [comment]
I have a series of commits to enable ICESUGAR, but it never really worked - not sure if if was the flash chip or something in my memory map. Should give it another shot, or prepare a pull request if you want to try it out.

**Comment by abderraouf-adjal** [comment]
I had a luck running a laggy UART hello word from old version using the
firmware.hex.

New versions of this repo is broken for icesugar, managed to edit for
successful compile, but no luck to see it alive.

**Comment by abderraouf-adjal** [comment]
@piso77 checkout #66 for icesugar-nano, it works fine in my end, I don't have an icesugar board to test on it, however it's not very different, just re-naming and make sure the SPI pins in .PCF file are correct by looking at the board schematic.

**Comment by piso77** [comment]
> @piso77 checkout #66 for icesugar-nano, it works fine in my end, I don't have an icesugar board to test on it, however it's not very different, just re-naming and make sure the SPI pins in .PCF file are correct by looking at the board schematic.

Thanks,  i'll give it a shot later today or tomorrow.
